go/chat/attachments/s3.go

```diff
@@ -87,7 +87,13 @@ func (a *S3Store) PutS3(ctx context.Context, r io.Reader, size int64, task *Uplo
 	region := a.regionFromParams(task.S3Params)
 	b := a.s3Conn(task.S3Signer, region, task.S3Params.AccessKey).Bucket(task.S3Params.Bucket)
 
-	if size <= minMultiSize {
+	multiPartUpload := size <= minMultiSize
```

Comment:
`multiPartUpload := size > minMultiSize` ???

Reply:
This is probably why the tests are failing, big mistake on my part, thanks!